app/components/sections/hero-section.tsx

```diff
@@ -154,6 +156,18 @@ function HeroSection({
               </H2>
             </motion.div>
           ) : null}
+          <motion.div variants={childVariants}>
+            <H4 as="p" variant="secondary" className="mt-3">
+              <Link
+                prefetch="intent"
+                to="/rss.xml"
+                className="text-secondary underlined hover:text-team-current focus:text-team-current"
+              >
+                {`Prefer RSS? Subscribe to my feed here.`}{' '}
+                <RssIcon className="inline-block" />
+              </Link>
+            </H4>
+          </motion.div>
```

Comment:
As is probably obvious this was just copy pasted from the components around it. I don't know if this is an ideal location for it, but personally I love when blogs have their RSS feed advertised right on the page. I'll gladly move it if needed though.